frontend/packages/knative-plugin/src/topology/knative-topology-utils.ts

```diff
@@ -485,8 +483,8 @@ export const getSinkUriTopologyEdgeItems = (
 };
 
 const getSinkTargetUid = (nodeData: NodeModel[], sinkUri: string) => {
-  const sinkNodeData = _.find(nodeData, ({ data: nodeResData }) => {
-    return sinkUri === nodeResData.data?.sinkUri;
+  const sinkNodeData = _.find(nodeData, ({ id }) => {
+    return sinkUri === id;
```

Comment:
You're comparing against the ID which is currently set as an encoded URI. I suspect this will fail with some URIs now.

I think what you had previously made more sense. Relying on comparison to the sinkUri in the data.